tests/integration/iop-externalistiod-remote-integration-test-defaults.yaml

```diff
@@ -25,3 +25,5 @@ spec:
   values:
     global:
       externalIstiod: true
+    telemetry:
+      enabled: false
```

Comment:
This shouldn't be needed, because it's the default (https://github.com/istio/istio/blob/790d9eef8badd0c9a70caf3676c9c06fc1571ca5/manifests/charts/istiod-remote/values.yaml#L119), but without it the install fails while trying to apply the EnvoyFilters because the Istio CRDs are not defined in a remote cluster.

^^^ @howardjohn Any idea what I'm missing?

Reply:
I am pretty sure `profiles/default.yaml` has precedence over values.yaml

Reply:
Hmm, since `profiles/default.yaml` explicitly sets every setting, what can be changed in values.yaml? This also makes me wonder if we need to explicitly set enabled=false in `profiles/external.yaml`. I guess it's not a bad idea to set it there, whether or not it's needed, just for clarity.

Reply:
`profiles/external.yaml` > `profiles/default.yaml` > `values.yaml`
so set the `enabled=false` in `profiles/external.yaml` makes sense, IMO.